Sources/SwipeableTabBarController.swift

```diff
@@ -29,7 +29,7 @@ open class SwipeableTabBarController: UITabBarController {
     
     /// Pan gesture for the swiping interaction
     //swiftlint:disable next implicitly_unwrapped_optional
-    private var panGestureRecognizer: UIPanGestureRecognizer!
+    private var panGestureRecognizer: UIPanGestureRecognizer?
```

Comment:
Need to change away from IOU since it's not safe and it's crashing if you set `selectedIndex` too early 🤷‍♂️